src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -891,7 +891,7 @@ internal List<Hex> GetIslandHexes()
         // These are all of the existing hexes, that if there are no islands will all be visited
         var shouldBeVisited = organelles.Select(p => p.Position).ToList();
```

Comment:
Isn't this method also now a duplicate? This one shouldn't exist, instead the one in organelle layout should be used.

Reply:
Forgot to stage the removal of the methods